src/Mod/Path/PathScripts/PathCircularHoleBase.py

```diff
@@ -778,10 +730,11 @@ def applyRotationalAnalysis(self, obj, base, angle, axis, subCount):
         elif axis == 'Y':
             vect = FreeCAD.Vector(0, 1, 0)
 
-        if obj.InverseAngle is True:
-            angle = -1 * angle
-            if math.fabs(angle) == 0.0:
-                angle = 0.0
+        # Commented out to fix PocketShape InverseAngle rotation problem
+        # if obj.InverseAngle is True:
+        #    angle = -1 * angle
+        #    if math.fabs(angle) == 0.0:
+        #        angle = 0.0
```

Comment:
can this be removed? is there any value in keeping the comment?

Reply:
Done - deleted.